src/assets/translations/en-US.json

```
@@ -3340,6 +3340,7 @@
       "This shows all the notifications.": "This shows all the notifications.",
       "This will navigate the user to the dashboard component.": "This will navigate the user to the dashboard component.",
       "This will navigate the user to the searched activity.": "This will navigate the user to the searched activity.",
+      "To": "to",
```

Biome (2.1.2)

[error] The key To was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)